avocado/plugins/diff.py

```diff
@@ -415,7 +415,10 @@ def _get_sysinfo(resultsdir, pre_post):
                 name_header = ['\n', '** %s **\n' % name]
                 sysinfo.extend(name_header)
                 with open(os.path.join(path, name), 'r') as sysinfo_file:
-                    sysinfo.extend(sysinfo_file.readlines())
+                    try:
+                        sysinfo.extend(sysinfo_file.readlines())
+                    except UnicodeDecodeError:
+                        continue
 
```

Comment:
What do you think about adding a debug/warning message?

Reply:
If we add the message in this exception we will have one warning in the beggining of the report for every time Diff finds the file, like:

```
[wrampazz@wrampazz avocado.dev]$ avocado diff 2656b041e90aaed49d4d730339fb7f51a9dbe173 8f137cf9ddae547ca72c572656e093ff1242d091 | head
Ignoring file journalctl.gz as it cannot be decoded.
Ignoring file random as it cannot be decoded.
Ignoring file journalctl.gz as it cannot be decoded.
Ignoring file random as it cannot be decoded.
```

I would prefer to have one single message at the end of the report, but we would need some refactor here to make it possible.

What do you think?

Reply:
If it is as log.DEBUG() and not displayed as default, I can't see a problem for each one. But this is up to you, just a suggestion.

Reply:
Humm, I tried `LOG_UI.debug` and it still shows the messages by default here.